build_tools/cmake/static_linker_test.c.in

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 The IREE Authors
```

Comment:
Not sure this is the best place to put the template. Happy to move elsewhere.

Reply:
Seems fine here for now